lib/internal/bootstrap/realm.js

```diff
@@ -285,6 +285,20 @@ class BuiltinModule {
     return canBeRequiredByUsersWithoutSchemeList.has(id);
   }
 
+  static normalizeRequirableId(id) {
+    let normalizedId = id;
+    if (StringPrototypeStartsWith(id, 'node:')) {
+      normalizedId = StringPrototypeSlice(id, 5);
+    }
+
+    if (!BuiltinModule.canBeRequiredByUsers(normalizedId) ||
+        (id === normalizedId && !BuiltinModule.canBeRequiredWithoutScheme(normalizedId))) {
```

Comment:
Just my personal curiosity, why do we do this check: `id === normalizedId` ?

Reply:
@tony-go it's a way to check if the `id` has the `node:` scheme. If the scheme was present, it would have been removed in the `normalizedId` string, so the value of `normalizedId` would be different from `id`.

Reply:
yeah, make sense in the end. thanks @RaisinTen ❤️